### PR TITLE
Update Build_all.yml

### PR DESCRIPTION
**Pull request by mathrack**
Update the CI script to restore the default flags

### CODE REVIEW

File: .github/workflows/Build_all.yml [hunk 1]
```diff
@@ -145,17 +145,17 @@ jobs:
         source /opt/intel/oneapi/setvars.sh
         export FC=mpiifort
```
Comment (pbartholomew08) [inline .github/workflows/Build_all.yml:146]: also `rm -rf ./build`?

File: .github/workflows/Build_all.yml [hunk 2]
```diff
@@ -178,11 +178,11 @@ jobs:
         export FC=mpif90
```
Comment (pbartholomew08) [inline .github/workflows/Build_all.yml:178]: also `rm -rf ./build`?

Comment (mathrack) [reply .github/workflows/Build_all.yml:178]: For the first build of the step, no cleaning is needed

File: .github/workflows/Build_all.yml (hunk 1 above)
Comment (mathrack) [reply .github/workflows/Build_all.yml:146]: For the first build of the step, no cleaning is needed

**Comment by pbartholomew08** [comment]
ok, wasn't sure if they necessarily ran in order

**Review by pbartholomew08 — APPROVED** [review]

**Pull request merged**